packages/training-link/src/TrainingLink.tsx

```diff
@@ -5,19 +5,16 @@ const shouldProbablyBeInUIKit: CSSProperties = { paddingTop: '12px' };
 const linkStyles: CSSProperties = { fontWeight: 'bold' };
 
 export type TrainingLinkProps = {
-  /**Link to training video.*/
+  /** Link to training video. */
   link?: string;
-  /**Name of your app to make the sentence complete.*/
-  name?: string;
 };
 
-const TrainingLink = ({ name, link }: TrainingLinkProps): JSX.Element => (
+const TrainingLink = ({ link }: TrainingLinkProps): JSX.Element => (
   <span style={shouldProbablyBeInUIKit} className="ml-auto">
     Need Help?{' '}
     <a href={link} style={linkStyles} className="link" target="_blank" rel="noopener noreferrer">
-      Watch a demo <span className="sr-only">(opens in a new tab)</span>
-    </a>{' '}
-    for {name}
+      Learn More
```

Comment:
This was done as part of POCL-5217: CDQ4 | ECS | Request Platform Update for Learn More Link
According to my PO the platform team requested this change. they want a simple learn more link instead of watch a view with the application name

Reply:
We shouldn't completely remove the old text. I would instead refactor so if there's no name passed then it would read `Learn More`, otherwise keep the `Watch a Demo for ...` format.

Reply:
@LauRoxx  I have updated the pull request. Thanks

Reply:
@LauRoxx updated the PR again